client/layout/sidebar/index.jsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+export default React.createClass( {
+	displayName: 'Sidebar',
+
+	propTypes: {
+		className: React.PropTypes.string,
+		onClick: React.PropTypes.func
+	},
+
+	render: function() {
+		return (
+			<ul className="sidebar" onClick={ this.props.onClick }>
```

Comment:
I find it a bit odd that we render this as a `<ul />`, especially when many of its children aren't `<li>` but rather header elements. Should we use [`<nav />`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) instead or consider wrapping the headers in `<li />`?